apps/opik-backend/src/test/java/com/comet/opik/api/resources/v1/priv/TracesResourceTest.java

```diff
@@ -3224,6 +3224,44 @@ void getTraceWithUsage() {
             getAndAssert(trace, projectId, API_KEY, TEST_WORKSPACE);
         }
 
+        @ParameterizedTest
+        @MethodSource
+        void getTraceWithCost(BigDecimal spanExpectedCost, String model) {
+            BigDecimal traceExpectedCost = spanExpectedCost == null ? null : spanExpectedCost.multiply(BigDecimal.valueOf(5));
+            var projectName = RandomStringUtils.randomAlphanumeric(10);
+            var trace = factory.manufacturePojo(Trace.class)
+                    .toBuilder()
+                    .id(null)
+                    .projectName(projectName)
+                    .usage(Map.of("completion_tokens", 200 * 5L, "prompt_tokens", 300 * 5L, "total_tokens", 4 * 5L))
```

Comment:
Can we generate random values to make the test more robust?

Reply:
No we can't. It should be related to spans to assert and also usage keys should be specific for cost calculation and can't be random.

Reply:
I updated per your request. But could you please explain how hardcoded usage values might introduce flakiness?